src/admin/resolvers/queries/Collection.ts

```diff
@@ -49,9 +49,15 @@ export async function searchCollections(
     throw new ForbiddenError(ACCESS_DENIED_ERROR);
   }
 
-  if (!filters || (!filters.author && !filters.title && !filters.status)) {
+  if (
+    !filters ||
+    (!filters.author &&
+      !filters.title &&
+      !filters.status &&
+      !filters.labelExternalIds)
+  ) {
```

Comment:
thoughts on pulling this out into a helper function?

Reply:
If you do a lot of validation like this, might be worthwhile to consider something like Joi to model/validate (potentially combined with graphql schema constraints?). https://joi.dev/api/?v=17.7.0#objectorpeers-options

Reply:
Also... another reason I hate Javascript and its missing set operations... This could alternatively be a set intersection of length > 1, rather than multiple conditionals.

Reply:
> thoughts on pulling this out into a helper function?

if this get repeated in other queries, seems worth it